app/helpers/setup_helper.rb

```diff
@@ -4,8 +4,11 @@
 # SetupHelper contains all the setup view helpers.
 module SetupHelper
   # https://www.iso.org/iso-3166-country-codes.html
-  # taken from https://coderwall.com/p/xww5mq/two-letter-country-code-regex
-  def country_code_regex
-    YAML.load_file(Rails.root.join("config", "country_codes.yml"))["iso"][3166]["regex"]
+  def country_codes
+    YAML.load_file(
+      Rails.root.join("config", "country_codes.yml")
+    )["iso"][3166]["codes"].collect do |c|
```

Comment:
I think if you remove the dashes in front of the country codes in the yaml file, you will get a Hash instead of an Array of (single element) Hashes. Then you can simply call `to_a` here instead of `collect` to get the same result:

```
 YAML.load_file(Rails.root.join("config", "country_codes.yml")["iso"][3166]["codes"].to_a
```